shelley/chain-and-ledger/dependencies/non-integer/doc/non-integer-calculations.tex

```diff
@@ -438,23 +490,24 @@ \section{Optimisations for Specific Use-Cases}
     \sum_{n=0}^k\frac{x^{n}}{n!} + R_{k}(x)
 \end{equation*}
 
-Where using an upper bound $M$ on the domain of $x$, the remainder (or error
+By using an upper bound $M$ on the domain of $x$, the remainder (or error
 term) $R_{k}(x)$ can be estimated as follows:
 
 \begin{equation*}
   R_{k}(x) \leq \vert M \cdot \frac{x^{k+1}}{(k+1)!}\vert
 \end{equation*}
 
-For the use-case of the leader election, a good integral bound for the error
+For the leader election use-case, a good integral bound for the error
 estimation is the maximal value of $\vert\ln(1 - f)\vert\leq M$ for
-$f \in [0, 0.95]$ is $M = 3$. For any larger value of $f$, a different value
-would have to be chosen. The current value for $f$ is $0.1$.
-
-In general, for this use case one can easily estimate $M$, as the smallest
-integer larger than the exponential function evaluated at the upper bound of the
-interval of the domain of $x$. For each partial sum $\Sigma_{k}$, one can then
-test whether $p$ is greater than $\Sigma_k + \vert R_{k}(x) \vert$ or less than
-$\Sigma_k - \vert R_{k}(x)\vert$ to decide whether one can stop at an early
+$f \in [0, 0.95]$, that is $M = 3$\khcomment{check!}.
```

Comment:
the good news is that, because of `f` being the active slot coefficient, we will very likely stay at lower values. And IIRC then `M=12` would be enough for `[0, 0.99]`

Reply:
Great!